relayer/cmd/import_beacon_state.go

```diff
@@ -92,13 +92,13 @@ func importBeaconState(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("read finalized state data from file: %w", err)
 	}
 
-	afterDenebFork := (conf.Source.Beacon.Spec.ForkVersions.Deneb + 1) * 32
+	afterElectraFork := (conf.Source.Beacon.Spec.ForkVersions.Electra + 1) * 32
 
-	attestedState, err := syncer.UnmarshalBeaconState(afterDenebFork, attestedData)
+	attestedState, err := syncer.UnmarshalBeaconState(afterElectraFork, attestedData)
```

Comment:
So for the testnet, should it be `afterFuluFork`? Could we just use the latest block on chain instead?

Reply:
Good point, added in https://github.com/Snowfork/snowbridge/pull/1623/commits/15939edbae94be6abcefba8db4e6daca14362d3b